datafusion/core/Cargo.toml

```diff
@@ -85,6 +87,7 @@ sqlparser = "0.17"
 tempfile = "3"
 tokio = { version = "1.0", features = ["macros", "rt", "rt-multi-thread", "sync", "fs", "parking_lot"] }
 tokio-stream = "0.1"
+url = "2.2"
```

Comment:
This is a new dependency, it is brought in by hyper and friends and so again is pretty ubiquitous